index.js

```diff
@@ -49,15 +49,15 @@
         nextSibling = el.nextElementSibling;
         traverse(el);
         if(el.nodeName == 'SPAN') {
-          if (el.style.fontWeight == 'bold') {
-            if(el.style.fontStyle == 'italic'){
+          if (el.style.fontWeight === 'bold' || el.style.fontWeight === '700') {
```

Comment:
Found this great gif when searching "truth"
![](https://media.giphy.com/media/sJtaW5KrzLf5S/giphy.gif)